Frontend/ui-customer/src/Components/QuoteListModel/View.purs

```diff
@@ -233,7 +234,7 @@ findingRidesView state push =
       lottieAnimationView
       [ id (getNewIDWithTag "lottieLoaderAnim")
       , afterRender (\action-> do
-                    void $ pure $ startLottieProcess lottieAnimationConfig{ rawJson = (getAssetsBaseUrl FunctionCall) <> "lottie/finding_rides_loader_with_text.json", lottieId = (getNewIDWithTag "lottieLoaderAnim") }
+                    void $ pure $ startLottieProcess lottieAnimationConfig{ rawJson = if (getMerchant FunctionCall == YATRI && state.vehicleVariant == "AUTO_RICKSHAW") then (getAssetsBaseUrl FunctionCall) <> "lottie/finding_rides_loader_with_text_auto.json" else (getAssetsBaseUrl FunctionCall) <> "lottie/finding_rides_loader_with_text.json", lottieId = (getNewIDWithTag "lottieLoaderAnim") }
```

Comment:
can we put this also in config.

Reply:
Will do this changes and raise a separate PR for refactoring the vehicleType in driver and customer side